packages/addons/addon-depends/mtr/package.mk

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2019 Mateusz Gruszczynski (https://www.linxiarz.pl)
+
+PKG_NAME="mtr"
+PKG_VERSION="0.92"
+PKG_SHA256="f2979db9e2f41aa8e6574e7771767c9afe111d9213814eb47f5e1e71876e4382"
+PKG_LICENSE="GPL"
+PKG_SITE="ftp://ftp.bitwizard.nl/mtr"
```

Comment:
change to `PKG_SITE="http://bitwizard.nl/mtr"` - this is the location to read about the package, not the source location (which is in PKG_URL).